### ISSUE TITLE
How can I create square object using target move?

### ISSUE THREAD

**Issue by JellyCZYT** [SebLague/Chess-Challenge#433]

**Comment by SebLague** [comment]
I don’t quite understand the question, could you rephrase or give more info about what you’re trying to do?

**Comment by JellyCZYT** [comment]
I need to create a Square object but I can just create it using a move that is the last move in the draft board, but I can only make it using the algebraic name. How can I create it using the last move?
